src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoEncryptedKey.cs

```diff
@@ -32,14 +32,14 @@ public EncryptedKey EncryptedKey
         public override XmlElement GetXml()
         {
             if (_encryptedKey == null)
-                throw new CryptographicException(SecurityResources.GetResourceString("Cryptography_Xml_InvalidElement"), "KeyInfoEncryptedKey");
+                throw new CryptographicException(SR.Cryptography_Xml_InvalidElement, "KeyInfoEncryptedKey");
```

Comment:
Hmm, looking at most of the other places similar to this, I'm wondering if it may give readers the wrong idea - apart from this example, almost all other element names used with `SR.Cryptography_Xml_InvalidElement` do not correspond to class names. And if this class is ever renamed (ok, part of public API so unlikely), it would affect the XML element name used here.

What do you reckon? I can easily make this change if you still want it.